src/screens/SUE/SueReportScreen.tsx

```diff
@@ -0,0 +1,234 @@
+  serviceCode: string;
+};
+
+export const SueReportScreen = ({ navigation }: { navigation: any }) => {
+  const { data, loading } = useStaticContent({
+    refreshTimeKey: 'publicJsonFile-sueReportProgress',
+    name: 'sueReportProgress',
+    type: 'json'
+  });
+
+  const [currentPage, setCurrentPage] = useState(0);
+  const [serviceCode, setServiceCode] = useState<string>();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [selectedPosition, setSelectedPosition] = useState<Location.LocationObjectCoords>();
+
+  const scrollViewRef = useRef(null);
+
+  const {
+    control,
+    formState: { errors },
+    handleSubmit
+  } = useForm({
+    mode: 'onBlur',
+    defaultValues: {
+      city: '',
+      description: '',
+      email: '',
+      firstName: '',
+      homeNumber: '',
+      images: '[]',
+      lastName: '',
+      phone: '',
+      street: '',
+      termsOfService: false,
+      title: '',
+      zipCode: ''
+    }
+  });
+
+  const { mutateAsync } = useMutation(postRequests);
+
+  const onSubmit = async (sueReportData: TReports) => {
+    if (!sueReportData.termsOfService) {
+      return Alert.alert(texts.defectReport.alerts.hint, texts.defectReport.alerts.termsOfService);
+    }
+
+    if (
+      !sueReportData.street ||
+      !sueReportData.homeNumber ||
+      !sueReportData.zipCode ||
+      !sueReportData.city
+    ) {
+      Alert.alert(texts.defectReport.alerts.hint, texts.defectReport.alerts.error);
+
+      setIsLoading(false);
+      return;
+    }
+
+    const addressString = `${sueReportData.street}; ${sueReportData.homeNumber}; ${sueReportData.zipCode}; ${sueReportData.city}`;
+
+    const formData = {
+      addressString,
+      lat: selectedPosition?.latitude,
+      long: selectedPosition?.longitude,
+      serviceCode,
+      ...sueReportData
+    };
+
+    mutateAsync(formData)
+      .then(() => {
+        navigation.goBack();
+        Alert.alert(
+          texts.defectReport.successScreen.header,
+          texts.defectReport.successScreen.entry
+        );
+      })
+      .catch(() => {
+        setIsLoading(false);
+
+        Alert.alert(texts.defectReport.alerts.hint, texts.defectReport.alerts.error);
+        return;
+      });
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < data.length - 1) {
+      setCurrentPage(currentPage + 1);
+      scrollViewRef?.current?.scrollTo({
+        x: device.width * (currentPage + 1),
+        y: 0,
+        animated: true
+      });
+    }
+  };
+
+  const handlePrevPage = () => {
+    if (currentPage > 0) {
+      setCurrentPage(currentPage - 1);
+      scrollViewRef?.current?.scrollTo({
+        x: device.width * (currentPage - 1),
+        y: 0,
+        animated: true
+      });
+    }
+  };
+
+  if (loading) {
+    return (
+      <LoadingContainer>
+        <ActivityIndicator color={colors.refreshControl} />
+      </LoadingContainer>
+    );
+  }
+
+  return (
+    <>
```

Comment:
`<SafeAreaViewFlex>` missing? or something else regarding keyboard avoiding stuff?

Reply:
Right! I'll change that.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/966/commits/2b603bd8f6f8b43bf449efd90fce208da3b2c23c